overturemaps/cli.py

```diff
@@ -99,13 +99,27 @@ def cli():
     type=click.Choice(get_all_overture_types()),
     required=True,
 )
+
+@click.option(
+    "-r",
+    "--release",
+    default=ALL_RELEASES[-1],
+    type=click.Choice(ALL_RELEASES),
+    required=False,
+)
+
 @click.option("--connect_timeout", required=False, type=int)
 @click.option("--request_timeout", required=False, type=int)
-def download(bbox, output_format, output, type_, connect_timeout, request_timeout):
+def download(bbox, output_format, output, type_, release, connect_timeout, request_timeout):
+    if release not in ALL_RELEASES:
```

Comment:
Based on release being part of the click.Choice above, this shouldn't be allowed. Since we also don't check type, probably could skip the check on release here.

Reply:
RIght. I just meant that we don't check whether the type field is valid here either because it's already been validated.